tests/unit/ml_handlers/test_auto_ts.py

```diff
@@ -0,0 +1,154 @@
+import time
+
+import pandas as pd
+import pytest
+from mindsdb_sql import parse_sql
+from ..executor_test_base import BaseExecutorTest
+
+
+class TestAuto_ts(BaseExecutorTest):
+    def run_sql(self, sql):
+        ret = self.command_executor.execute_command(parse_sql(sql, dialect="mindsdb"))
+        assert ret.error_code is None
+        if ret.data is not None:
+            columns = [
+                col.alias if col.alias is not None else col.name for col in ret.columns
+            ]
+            return pd.DataFrame(ret.data, columns=columns)
+
+    def wait_predictor(self, project, name):
+        # wait
+        done = False
+        for attempt in range(200):
+            ret = self.run_sql(f"select * from {project}.models where name='{name}'")
+            if not ret.empty:
+                if ret["STATUS"][0] == "complete":
+                    done = True
+                    break
+                elif ret["STATUS"][0] == "error":
+                    break
+            time.sleep(0.5)
+        if not done:
+            raise RuntimeError("predictor wasn't created")
+
+    def test_missing_ts_column(self):
+        # create project
+        self.run_sql("create database proj")
+
+        self.run_sql(
+            """
+                CREATE MODEL proj.auto_ts_missing_ts_column
+                FROM files
+                    (SELECT * FROM Sales_and_Marketing)
```

Comment:
Do these tests run correctly on a fresh install? I don't see the `Sales_and_Marketing` file being added in the PR, so I suspect these may fail.